tests/Fake/FakeResolveInfo.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace SilverStripe\GraphQL\Tests\Fake;
+
+use GraphQL\Type\Definition\FieldDefinition;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\ObjectType;
+use GraphQL\Type\Definition\Type;
+use GraphQL\Type\Schema;
+
+class FakeResolveInfo extends ResolveInfo
```

Comment:
```suggestion
class FakeResolveInfo extends ResolveInfo implements TestOnly
```
Also add `use` statement.

Reply:
> Why do we need a FakeResolveInfo class? I just want to be sure that it's okay to do this for tests without getting false positive results.

Between v0.12 and v14 of webonyx/graphql-php the constructor for ResolveInfo changed from an array, to 10 params that are mostly non-nullable.  For the majority of the tests we don't even need the ResolveInfo object for anything, it's just a non-optional argument.  So the FakeResolveInfo is just a nice way to fill in the 10 constructor arguments really.

Asset-admin, versioned and elemental already have existing FakeResolveInfo classes so I just copied what was done there.